### ISSUE TITLE
Allow Clipboard Manager to Paste Images

### ISSUE THREAD

**Issue by frozht** [futo-org/android-keyboard#157]
Allow copy/paste images like in GBoard.
Recently copied items should also be prioritized in the top bar when opening the keyboard, or icons bar (if nothing in clipboard manager), instead of having suggested words at the start.

Great work so far.

**Comment by codokie** [comment]
For reference I've tried to add clipboard images support for the Heliboard keyboard in [this](https://github.com/Helium314/HeliBoard/pull/722) pull request.

The FUTO keyboard implementation of this feature could be similar.

**Comment by Hugh288** [comment]
Just curious if this feature is still on the radar?
Right now I'm using Samsung edge panel to access the Samsung clipboard. 
Which works pretty well, but not as good as built in to the keyboard.

**Comment by dean-futo** [comment]
@Hugh288 yes, not a top priority right now but we will get to this eventually.

**Comment by ballo** [comment]
Wow, I guess I'm back to using gboard with all it's problems. Yes, I absolutely need to be able to paste images

**Comment by abb128** [comment]
Released in v0.1.28

**Comment by MulverineX** [comment]
@abb128 this does not work when a screenshot is taken, which is the primary reason most people wanted this feature. Other issues were closed in favor of this one, should a new issue be created or will this be re-opened?

It should be this simple to use:
1. You take a screenshot
2. It automatically appears at the top of your clipboard history in FUTO Keyboard
3. Can be pasted into any app that supports it including Discord

**Comment by abb128** [comment]
@MulverineX Did you enable the option for screenshots? It's a toggle that's off by default because you need to grant access to files for it

**Comment by MulverineX** [comment]
@abb128 the note in the changelog IMO could be clearer that it doesn't work until you manually grant the permission and toggle the feature on.

Additionally, all this does is make recent screenshots appear in your clipboard history, it does not use the latest screenshot as your clipboard; pressing the "Paste from Clipboard" action button doesn't work, you have to go to your clipboard history and select it from there.

The point here being, that when using Gboard, it takes one tap to paste a screenshot after taking it, this is ideal.

**Comment by TheBoxTrot** [comment]
> @MulverineX Did you enable the option for screenshots? It's a toggle that's off by default because you need to grant access to files for it

I've tried this and it still doesn't pick up on screenshots. 
I have a Samsung galaxy which allows you to switch the directory where screen shots are sent to. Which I thought is why but after moving it back to the dcim folder the keyboard still does not pick up new screenshots

**Comment by Zvonimir-FUTO** [comment]
@TheBoxTrot I have opened a separate issue to track this https://github.com/futo-org/android-keyboard/issues/2006
